### PR TITLE
feat(documentation): Set up documentation with Sphinx

### PR DESCRIPTION
**Pull request by labdala**
**Under construction**

**To do**:
 - Tweak sphinx and MyST versions to see if we can get md into rst file;
 - Set up Read the docs, add Matt as a maintainer, make it update with PR review;

### CODE REVIEW

**Comment by jcfr** [comment]
These links may be helpful to answer questions related to use of MyST & sphinx:
* https://github.com/BICCN/cell-locator/pull/177
* https://github.com/Slicer/Slicer/pull/5662

File: docs/source/conf.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'itk-viewer-bootstrap-ui'
+copyright = '2022, Laryssa Abdala'
+author = 'Laryssa Abdala'
```
Comment (labdala) [inline docs/source/conf.py:24]: @thewtex , I thought referred to the website's author and realize this should be under your name?

Comment (thewtex) [reply docs/source/conf.py:24]: @labdala to reflect the collaborative effort, you can put: *Insight Software Consortium*

**Comment by labdala** [comment]
> These links may be helpful to answer questions related to use of MyST & sphinx:
> 
> * [Support generating sphinx doc BICCN/cell-locator#177](https://github.com/BICCN/cell-locator/pull/177)
> * [ENH: Add automatically generated CLI module documentation Slicer/Slicer#5662](https://github.com/Slicer/Slicer/pull/5662)

Thank you very much JC, these are great!!

**Review by thewtex — APPROVED** [review]

**Pull request merged**